### PR TITLE
Fixed explanation to make the concept clearer

### PR DESCRIPTION
**Pull request by jaintj95**
Earlier explanation was ambiguous and did not explain the concept of abstract classes properly. I have updated the explanation and made it clearer.

Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/java/abstract-class/index.md [hunk 1]
```diff
@@ -4,38 +4,45 @@ title: Abstract Classes in Java
 
 Lets discuss abstract classes. Before diving into this tutorial it is better that you have understood concepts of classes
```
Comment (amandeepmittal) [inline src/pages/java/abstract-class/index.md]: Make this one complete sentence instead of adding a new line character in between.

Comment (jaintj95) [reply src/pages/java/abstract-class/index.md]: Sorry, I am a little confused as to where you are asking me to make a change. I did not touch this part and left it as it was from the previous commit. Are you telling me that I should remove the new line at 4 and move the content at line 5 to line 4?

**Comment by amandeepmittal** [comment]
Thanks for your contribution 👍 @jaintj95.

File: src/pages/java/abstract-class/index.md (hunk 1 above)
Comment (amandeepmittal) [reply src/pages/java/abstract-class/index.md]: Yeah, moving the content from 4 to 5 but it seem resolved as of now.

**Pull request merged**